api/src/paths/security/persecution-harm/apply.ts

```diff
@@ -94,15 +95,21 @@ export function applySecurityRulesToArtifacts(): RequestHandler {
       await connection.open();
 
       const securityService = new SecurityService(connection);
+      const artifactService = new ArtifactService(connection);
 
+      // Remove all security rules from artifacts.
```

Comment:
Now or in future: We probably will want to be more fine-grain with this. For example: If an update to security rules is sent, it is better if we can selectively add the new rules, remove the missing rules, and leave untouched any rules that didn't change. Rather than blindly removing all existing rules and adding all incoming rules as net new.

Its more annoying to code it this way, but technically it makes the auditing more accurate

Reply:
Sounds good, have time this sprint to include that.

Reply:
We do this sort of thing in the SIMS survey-service updateSurveyPermitData() function